brewtils/schema_parser.py

```diff
@@ -300,6 +300,35 @@ def parse_job(cls, job, from_string=False, **kwargs):
         """
         return cls.parse(job, brewtils.models.Job, from_string=from_string, **kwargs)
 
+    @classmethod
+    def parse_job_ids(cls, job_id_list, from_string=False, **kwargs):
+        """Convert raw JSON string or list of strings to a list of job ids.
+
+        Passes a list of strings through unaltered if from_string is False.
+
+        Args:
+            job_id_list: Raw input
+            from_string: True if input is a JSON string, False otherwise
+            **kwargs: Additional parameters to be passed to the Schema (e.g. many=True)
+
+        Returns:
+            A list of job ids.
+        """
+        # this is needed by easy_client
```

Comment:
I might need help understanding why exactly this is needed.

Reply:
Because the action methods in EasyClient objects that return data are wrapped in a `@wrap_response` decorator. That decorator needs as its first argument a function that translates JSON back to a python object, a brewtils model in pretty much every other case. But since we got rid of brewtils model objects for the new code, this is a workaround that still maintains some minimum *semblance* of consistency. Otherwise we have a dozen functions with similar and parallel implementations and one outlier.

A better solution is to rewrite the whole thing to not do it that way, since assuming that easy client will only ever manipulate brewtils objects (with minor exceptions) isn't playing out as true. But that's an issue for another ticket and a timeline long down the road.